workbench-core/base/src/aws/helpers/dynamoDB/dynamoDBService.ts

```diff
@@ -140,6 +142,29 @@ export default class DynamoDBService {
     return getter;
   }
 
+  /**
+   * Queries the DynamoDB table.
+   *
+   * @param params - optional object of optional properties to generate a query request
+   * @returns Promise<PaginatedQueryResponse>
+   *
+   * @example Use this to get paginated items from the DynamoDb table.
+   * ```ts
+   * const result = dynamoDBService.getPaginatedItems({sortKey: 'value', eq: {N: '5'}});
+   * ```
+   */
+  public async getPaginatedItems(params?: QueryParams): Promise<PaginatedQueryResponse> {
```

Comment:
Nitpick: Thoughts on renaming `PaginatedQueryResponse` to `PaginatedItemResponse`? I think the name of the method and the response type should be similar so developers can relate easily relate the two concepts. Also the `PaginatedItemResponse` doesn't have to be from a query call.

Reply:
I would also prefer this method to explicitly mention query since it is the wrapper for `query()`. Why did you change it from `executeQuery()`?

Reply:
@maghirardelli I felt like I could better describe what the method was doing with this name over `executeQuery`, which doesn't give any indication that the you're potentially getting back more than one item and that pagination could be involved.